railties/lib/rails/generators/named_base.rb

```diff
@@ -150,23 +150,23 @@ def redirect_resource_name # :doc:
         def model_resource_name(prefix: "") # :doc:
           resource_name = "#{prefix}#{singular_table_name}"
           if options[:model_name]
-            "[#{controller_class_path.map { |name| ":" + name }.join(", ")}, #{resource_name}]"
+            "[#{(controller_class_path.map { |name| ":" + name }.to_a << resource_name) * ', '}]"
```

Comment:
Why us `to_a` being called here?

Reply:
If all tests pass, your changes are good.

Reply:
Sorry I didn't get it. Why is the `to_a` being called here? `map` already returns an array, doesn't it?

Reply:
You're right. It's not needed.